### DISCUSSION TITLE
error Connection errored out.

### DISCUSSION THREAD

**Discussion by niuzr2023** [mikumifa/cppTickerBuy#22] [提问专区]
输入连接出现error Connection errored out. Main函数下面出现报错 
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "D:\pythonProject\.venv\cppTickerBuy-main\Lib\site-packages\pydantic\type_adapter.py", line 270, in _init_core_attrs
    self._core_schema = _getattr_no_parents(self._type, '__pydantic_core_schema__')
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\pythonProject\.venv\cppTickerBuy-main\Lib\site-packages\pydantic\type_adapter.py", line 112, in _getattr_no_parents
    raise AttributeError(attribute)
AttributeError: __pydantic_core_schema__

During handling of the above exception, another exception occurred:

**Comment by mikumifa** [comment]
重新，关梯子

**Reply by mikumifa** [reply]
重启

**Reply by niuzr2023** [reply]
问题还在诶 输入链接之后网站显示network问题pycharm输出中报了很多错

**Reply by mikumifa** [reply]
我没见过
会pycharm的话这玩意应该好解决吧
去网上搜

**Reply by mikumifa** [reply]
发挥程序员的主观能动性

**Reply by niuzr2023** [reply]
主要是用大佬的bili抢票脚本就没有这个问题

**Reply by niuzr2023** [reply]
解决了 requirement外面重新安装了下playwright 谢谢大佬
